physical_property/logP/analysis/SAMPL7-user-map-logP.csv

```diff
@@ -0,0 +1,37 @@
+11,10006,004AF,004AF-1559-logP-ensemble_logP_model-1.csv,deadline,Jae,Shin,NetTargets Inc.
+12,10006,2DF02,2DF02-1559-logP-ensemble_logP_model-2.csv,deadline,Jae,Shin,NetTargets Inc.
+13,10007,08DA5,08DA5-1559-logP-EvrimArslan-1.csv,deadline,Evrim,Arslan,Bogazici University
+14,10008,02F1F,02F1F-1559-logP-DB-1.csv,deadline,Alex,Dickson,Michigan State University
+15,10008,4K631,4K631-1559-logP-DB-2.csv,deadline,Alex,Dickson,Michigan State University
+16,10008,43M66,43M66-1559-logP-DB-3.csv,deadline,Alex,Dickson,Michigan State University
+17,10008,847L9,847L9-1559-logP-DB-4.csv,deadline,Alex,Dickson,Michigan State University
+18,10009,0357F,0357F-1559-logP-IEFPCMMST-1.csv,deadline,CBDD,Group,University of Barcelona and University of Costa Rica
+19,10010,047D4,047D4-1559-logP-ECRISM-1.csv,deadline,Stefan,Kast,TU Dortmund University
+20,10011,D4406,D4406-1559-logP-AndrewPaluch_MD-1.csv,deadline,Andrew,Paluch,Miami University
+21,10011,065C7,065C7-1559-logP-AndrewPaluch_MD-2.csv,deadline,Andrew,Paluch,Miami University
+22,10012,07B59,07B59-1559-logP-PieroProcacci_NES1_B-1.csv,deadline,Piero,Procacci,Universita' di Firenze
+23,10012,F46P0,F46P0-1559-logP-PieroProcacci_NES1_G-2.csv,deadline,Piero,Procacci,Universita' di Firenze
+24,10012,C4A03,C4A03-1559-logP-PieroProcacci_NES1_J-3.csv,deadline,Piero,Procacci,Universita' di Firenze
+25,10013,09F0E,09F0E-1559-logP-DavyGuan-1.csv,deadline,Davy,Guan,The University of Sydney
+27,10014,02BF8,02BF8-1559-logP-RodriguezPaluch_SM12_1-1.csv,deadline,Sergio,Rodriguez,"University of Santiago del Estero, Argentina"
+28,10014,006AC,006AC-1559-logP-RodriguezPaluch_SM8_1-2.csv,deadline,Sergio,Rodriguez,"University of Santiago del Estero, Argentina"
+29,10014,7RS67,7RS67-1559-logP-RodriguezPaluch_SM8_2-3.csv,deadline,Sergio,Rodriguez,"University of Santiago del Estero, Argentina"
+30,10014,0A274,0A274-1559-logP-RodriguezPaluch_SMD_1-4.csv,deadline,Sergio,Rodriguez,"University of Santiago del Estero, Argentina"
+31,10014,085FC,085FC-1559-logP-RodriguezPaluch_SMD_2-5.csv,deadline,Sergio,Rodriguez,"University of Santiago del Estero, Argentina"
+32,10015,5W956,5W956-1559-logP-MLRUCR-1.csv,deadline,CB3,Group,University of Costa Rica
+33,10016,0636A,0636A-1559-logP-nhlbi-1.csv,deadline,Michael,Jones,NIH NHLBI
+34,10016,03F9F,03F9F-1559-logP-nhlbi-2.csv,deadline,Samar,Prasad,NIH NHLBI
+35,10017,6Q594,6Q594-1559-logP-prediction_Iorga_Beckstein_LigParGen-1.csv,deadline,Bogdan,Iorga,"ICSN, CNRS, Gif-sur-Yvette, France/Arizona State University, USA"
+36,10017,0T68C,0T68C-1559-logP-prediction_Iorga_Beckstein_CGenFF-2.csv,deadline,Bogdan,Iorga,"ICSN, CNRS, Gif-sur-Yvette, France/Arizona State University, USA"
+37,10017,043B8,043B8-1559-logP-prediction_Iorga_Beckstein_GAFF-3.csv,deadline,Bogdan,Iorga,"ICSN, CNRS, Gif-sur-Yvette, France/Arizona State University, USA"
+38,10017,0B909,0B909-1559-logP-prediction_Iorga_Beckstein_OPLS_AA-4.csv,deadline,Bogdan,Iorga,"ICSN, CNRS, Gif-sur-Yvette, France/Arizona State University, USA"

```

Comment:
I think these map files may be entirely unnecessary; see changes I made for host-guest and protein-ligand challenges.

Reply:
I removed the map file.